src/styles/blocks/process.scss

```diff
@@ -0,0 +1,316 @@
+.process {
+  &__description {
+    @extend %openSansSmallContent;
+    color: $gray-color;
+  }
+
+  &__cards {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+
+    @include onTablet {
+      gap: 30px;
+    }
+
+    @include onSmallDesktop {
+      margin-top: 140px;
+      gap: 20px;
+      grid-template-columns: repeat(auto-fill, 214px);
+    }
+
+    @include theDesign {
+      gap: 30px;
+      grid-template-columns: repeat(auto-fill, 264px);
+    }
+    &-bottom {
+      margin-bottom: 24px;
+
+      @include onTablet {
+        margin-bottom: 0;
+      }
+    }
+
+    &-top {
+      @include onTablet {
+        transform: translateY(-132px);
+      }
+
+      @include onSmallDesktop {
+        transform: translateY(-140px);
+      }
+
+      @include onDesktop {
+        transform: translateY(-132px);
+      }
+    }
+  }
+
+  &__card {
+    background-color: $white-color;
+    box-shadow: 0 9px 18px rgba(37, 41, 49, 0.03);
+    border-radius: 16px;
+    padding: 32px;
+    transition: all $transition-duration;
+
+    &:hover {
+      transform: scale(1.1) translateY(-50%);
+      box-shadow: 0 0 20px $dark-blue-color;
+    }
+
+    &--blue {
```

Comment:
Are you using this modifiers? If not please remove redundant styles

Reply:
There are different variants, as in Figma design